src/Kafka/CreateKafkaInstance/components/FieldSize.tsx

```diff
@@ -11,7 +11,7 @@ import { FieldSizeHelperTextTrial } from "./FieldSizeHelperTextTrial";
 export type FieldSizeProps = {
   value: number;
   sizes: Size[] | undefined;
-  remainingQuota: number;
+  remainingQuota: number | undefined;
```

Comment:
will it possible that remainingQuota can be undefined ? I think it always should be a number.

Reply:
This is to turn off the alerts in the size field about the over-quota scenario, since the error is instead displayed into the billing field.